xml/script/process.js

```diff
@@ -370,6 +370,9 @@ function passThree(schemaFilename) {
         if (systemId == "http://www.w3.org/2001/xml.xsd") {
             resolvedFilename = baseDir + "lib/external/xml.xsd";
         }
+        else if (systemId == "https://www.w3.org/2001/xml.xsd") {
```

Comment:
Should this also cover both http + https like the change at line 511?

`else if (systemId == "http://www.w3.org/2001/xml.xsd" || systemId == "https://www.w3.org/2001/xml.xsd") {`

Reply:
That's a fair point. I just did what was needed for the tests to pass. I'll try this update as well...